embedding/csharp/Tizen.Flutter.Embedding/Interop/FlutterDesktopView.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright 2022 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+using System;
+using System.Runtime.InteropServices;
+
+namespace Tizen.Flutter.Embedding
+{
+    public class FlutterDesktopView : SafeHandle
+    {
+        public FlutterDesktopView() : base(IntPtr.Zero, true)
+        {
+        }
+
+        public override bool IsInvalid => handle == IntPtr.Zero;
+
+        public static implicit operator bool(FlutterDesktopView view)
+        {
+            return !view.IsInvalid;
+        }
```

Comment:
This operator overloading should not be necessary at this moment.

FYI) `FlutterDesktopEngine` objects are implicitly converted to `bool` to avoid double negation in `Debug.Assert` checks.

```csharp
// Bad
Debug.Assert(!Handle.IsInvalid);
// Good
Debug.Assert(Handle);
```

Reply:
Thank you, I removed it.